nengo_loihi/neurons.py

```diff
@@ -8,12 +8,11 @@
 import numpy as np
 
 try:
-    import nengo_dl
-    import nengo_dl.neuron_builders
     import tensorflow as tf
-except ImportError:  # pragma: no cover
-    nengo_dl = None
+    HAS_TF = True
+except ImportError:
```

Comment:
This does not appear to be covered. I think we want to keep the `pragma: no cover`.

Reply:
Removing the `pragma: no cover` was accidental, will add it back